fds/utils.py

```diff
@@ -144,36 +144,44 @@ def get_input_from_user(question: str, type: str = "input") -> str:
     return answer
 
 
-def get_expand_input_from_user(question: str, choices: List[Dict[str, str]], default: Union[str, bool, int]) -> str:
+def get_expand_input_from_user(question: str, choices: List[Dict[str, str]], default: Union[str, bool, int], show_detailed_options: bool) -> str:
```

Comment:
1. `show_detailed_options` is not used
2. Better to make it an optional parameter with default value `False`

Reply:
Fixed this

Reply:
but didnt make it optional parameter because i handled the places where this is used